### PR TITLE
chore(portal): add note for multi rg regarding plugins (TDX-2637)

### PR DESCRIPTION
**Pull request by davidma415**
### Summary

Adding a note that some plugins are not compatible if they are not in the default runtime group.

### CODE REVIEW

File: app/konnect/dev-portal/applications/enable-app-reg.md [hunk 1]
```diff
@@ -133,3 +133,7 @@ If you need to use a version of {{site.base_gateway}} less than 3.0, you must cr
 In the `default` runtime group, applications are still linked to {{site.base_gateway}} `consumers` and use the `acl` plugin to control access between an application's `consumer` and a service version. This configuration is deprecated. It is recommended to upgrade your runtime instances to {{site.base_gateway}} version 3.0+ to ensure future compatibility with the `konnect-application-auth` plugin, which has a built-in replacement for the `acl` plugin and doesn't rely on `consumers`.
 
 The `konnect-application-auth` plugin is used to manage access control and API key authentication for app registration and replaces the need for the `acl` and `key-auth` plugins. It is only supported in {{site.base_gateway}} 3.0+ and is used for app registration in every non-`default` runtime group.
+
+{:.note}
+> **Note:** If you are not in the `default` runtime group, consumer-based plugins (e.g. rate-limiting) are not compatible
```
Comment (Guaris) [inline app/konnect/dev-portal/applications/enable-app-reg.md:138]:
```suggestion
> **Note:** If you are not in the `default` runtime group, consumer-based plugins, like rate-limiting, are not compatible
```

**Comment by Guaris** [comment]
@davidma415 I noticed you added the Do Not Merge label. Is this waiting on other changes?

**Comment by acgoldsmith** [comment]
This is tied to DOCU-2803

**Comment by davidma415** [comment]
> @davidma415 I noticed you added the Do Not Merge label. Is this waiting on other changes? 

I am just awaiting final approval from my team for this! Thanks for the copy edit.

**Review by Mierenga — APPROVED** [review]
LGTM

**Comment by davidma415** [comment]
@Guaris we can merge!

**Comment by acgoldsmith** [comment]
@davidma415 The tech preview badge still needs to be removed.. delete {:.badge .alpha} from line 12.

**Review by acgoldsmith — APPROVED** [review]
LGTM

**Pull request merged**